app/routes.py

```diff
@@ -1,2 +1,101 @@
-from flask import Blueprint
+from app import db
+from app.models.planet import Planet
+from flask import Blueprint, jsonify, request
 
+planets_bp = Blueprint("planets_bp", __name__, url_prefix="/planets")
+
+
+@planets_bp.route("", methods=["POST", "GET"])
+def handle_planets():
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "name" not in request_body or "description" not in request_body:
+            return jsonify("Invalid request"), 404
+
+        new_planet = Planet(
+            name=request_body["name"],
+            description=request_body["description"]
+        )
+        db.session.add(new_planet)
+        db.session.commit()
+
+        return jsonify(f"Planet {new_planet.name} created successfully"), 201
```

Comment:
Consider returning information about the planet created (its dictionary structure) rather than just a message here. The caller would probably like to at least know the id of the created record. Returning a dictionary with the id and message as two separate parts can make it easy to get the id programmatically, as well as a potentially human friendly message.